slack_bolt/logger/messages.py

```diff
@@ -266,7 +267,26 @@ def warning_unhandled_request(  # type: ignore
         )
     if is_event(req.body):
         # @app.event
-        event_type = req.body.get("event", {}).get("type")
+        event = req.body.get("event", {})
+        event_type = event.get("type")
+        if is_function(req.body):
+            # @app.function
+            callback_id = event.get("function", {}).get("callback_id", "function_id")
+            return _build_unhandled_request_suggestion(
+                default_message,
+                f"""
+@app.function("{callback_id}")
+{'async ' if is_async else ''}def handle_{callback_id}_function(body, event, client, logger):
+    logger.info(body)
+    {'await ' if is_async else ''}client.api_call(
+        "functions.completeError",
```

Comment:
why having the error  handler only? Generating something like this would be even more developer-friendly:
```python
@app.function("{callback_id}")
def handle_{callback_id}_function_events(body, event, client, logger):
    try:
        # TODO: do something here
        outputs = {}
        client.api_call(
            "functions.completeSuccess",
            json={
                "function_execution_id": context.function_execution_id,
                "outputs": outputs,
            },
        )
    except Exception as e:
        error = f"Failed to handle a function request (error: {e})"
        client.api_call(
            "functions.completeError",
            json={
                "function_execution_id": context.function_execution_id,
                "error": error,
            },
        )
```